Sources/KlaviyoSwift/Models/Event.swift

```diff
@@ -10,6 +10,10 @@ import Foundation
 import KlaviyoCore
 
 public struct Event: Equatable {
+    public enum IAFProfileEvent: Equatable {
```

Comment:
I don't think this needs to be in the `KlaviyoSwift` I would keep it local to `KlaviyoUI`